ethcore/src/lib.rs

```diff
@@ -178,3 +176,5 @@ pub mod test_helpers;
 pub use executive::contract_address;
 pub use evm::CreateContractAddress;
 pub use trie::TrieSpec;
+/// Ethcore Result
+pub type EthcoreResult<T> = Result<T, types::errors::EthcoreError>;
```

Comment:
this type is already defined in [`ethcore/src/error.rs`](https://github.com/paritytech/parity-ethereum/pull/10880/files#diff-4342081aac4ee027883e1efc84bedea4R22), right?

Reply:
I must have forgot to remove `error.rs`. My bad.